src/components/cardheader/CardHeader.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardHeader = props => {
+  const { title, icon = null } = props;
+
+  return (
+    <div className="card-header">
+      <div className="card-header__l">
+        <div className="title">{title}</div>
+      </div>
+      {!icon ? (
+        ''
+      ) : (
+        <div className="card-header__r">
+          <img src={icon} alt="Edit" />
+        </div>
+      )}
+    </div>
+  );
+};
+
+CardHeader.propTypes = {
+  title: PropTypes.string,
+  icon: PropTypes.object | null,
+};
```

Comment:
add a new line after it

Reply:
[form-preview 3cc943e] form-preview:#2 - added new line and conditional rendering